crates/bevy_ecs/src/system/mod.rs

```diff
@@ -33,41 +25,33 @@
 //! # bevy_ecs::system::assert_is_system(update_score_system);
 //! ```
 //!
-//! # System ordering
-//!
-//! While the execution of systems is usually parallel and not deterministic, there are two
-//! ways to determine a certain degree of execution order:
+//! # System construction
 //!
-//! - **System Stages:** They determine hard execution synchronization boundaries inside of
-//!   which systems run in parallel by default.
-//! - **Labeling:** First, systems are labeled upon creation by calling `.label()`. Then,
-//!   methods such as `.before()` and `.after()` are appended to systems to determine
-//!   execution order in respect to other systems.
+//! Any function or closure whose parameters implement the [`SystemParam`] trait can be automatically
+//! converted into a system:
 //!
-//! # System parameter list
-//! Following is the complete list of accepted types as system parameters:
-//!
-//! - [`Query`]
-//! - [`Res`] and `Option<Res>`
-//! - [`ResMut`] and `Option<ResMut>`
+//! - [`Query<...>`](Query)
```

Comment:
I hate this list, please just yeet it.

Reply:
Will users have some other way of learning about the available system params?

Reply:
Yeah, the `SystemParam` link will list implementers. We should probably just call that out, rather than trying to duplicate it.